library/src/hcc_detail/hipsparse.cpp

```diff
@@ -4898,6 +4898,94 @@ hipsparseStatus_t hipsparseZhyb2csr(hipsparseHandle_t         handle,
     return HIPSPARSE_STATUS_SUCCESS;
 }
 
+hipsparseStatus_t hipsparseSnnz(hipsparseHandle_t         handle,
+                                hipsparseDirection_t      dirA,
+                                int                       m,
+                                int                       n,
+                                const hipsparseMatDescr_t descrA,
+                                const float*              A,
+                                int                       lda,
+                                int*                      nnzPerRowColumn,
+                                int*                      nnzTotalDevHostPtr)
+{
+    RETURN_IF_ROCSPARSE_ERROR(rocsparse_snnz((rocsparse_handle)handle,
+                                             (rocsparse_direction)dirA,
```

Comment:
should use a function for conversion (see e.g. how hipsparseOperation_t is converted), else this would result in failure if rocsparse or cusparse change the enums.

Reply:
done.